queries.sql

```diff
@@ -22,4 +22,23 @@ SELECT * from animals WHERE neutered = TRUE;
 SELECT * from animals WHERE name <> 'Gabumon';
 
 /*Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
-SELECT * from animals WHERE weight_kg >=10.4 AND weight_kg <=17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg >=10.4 AND weight_kg <=17.3;
+
+/* Count number of animals */
+SELECT COUNT(*) FROM animals;
+
+/* Count number of animals that have not attempted escape */
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+/* Avarage weight of animals */
+SELECT AVG(weight_kg) FROM animals;
+
+/* Sum escape attempts and compare between neutered and non-neutered */
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
+
+/* Minimum and maximum weights of each type of animal*/
+SELECT neutered, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY neutered;
```

Comment:
- [ ] This query should return the minimum and the maximum weight of each type of animal i.e based on the species of animal, however, it is returning minimum and maximum weight of animals based on neutered or not neutered. Kindly fix this by changing your query to `SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;`